docs/Documentation/run_RBM.md

```diff
@@ -1,66 +1,64 @@
+*Salmon_River_data/UH_Output* | the sample output from executing the routing model, **rout**, and the input data for the sample problem.
+*Salmon_River_data/VIC_Forcing* | the daily precipitation (mm), max/min air temperatures (°C), and wind speed (m/sec) that provides forcings for each **VIC** grid cell.
+*Salmon_River_data/run_VIC* | the snowbands, soil, veg-param files, the world_veg_lib files and example global parameter files
+*Salmon_River_data/VIC_output* | the meteorologic output files, full_data_lat_long, and the hydrologic output - files, flux_lat_long, from the VIC simulations.
 
 ## 2. Create the RBM Executable
-Navigate to the folder, **../src**, and type:
+Navigate to the folder, **RBM/src**, and type:
 <center>**make**</center>
 
 This will create the executable, RBM, and copy a version to the folder, **../Test_Case**, where the example problem is found.
 
 
 ## 3. Run the Model
 ### Step 1 Build the forcing function files (flow and meteorology)
-In the example problem, I used **VIC4.2.d** with the two global parameter files in the directory, **run_VIC**. To create the necessary files (executable file to run VIC), you need to download [VIC4.2.d](https://github.com/UW-Hydro/VIC/releases/tag/VIC.4.2.d). The instructions to generate the executable file **vicNl** and run VIC can be found [here](http://vic.readthedocs.io/en/vic.4.2.d/). Copy the executable file **vicNl** to the folder **run_VIC**. The example global files to generate baseflow, runoff and heat budgets are also directly included in the folder **run_VIC**, which you can use directly. The two were implemented as follows:
+In the example, we run hydrological model **VIC4.2.d** to generate flow and energy fluxes, which are input data for RBM. The example global parameter files can be found in the directory **run_VIC**. To create the necessary files (executable file to run VIC), you need to download [VIC4.2.d](https://github.com/UW-Hydro/VIC/releases/tag/VIC.4.2.d). The instructions to generate the executable file **vicNl** and run VIC can be found [here](http://vic.readthedocs.io/en/vic.4.2.d/). Copy the executable file **vicNl** to the folder **run_VIC**. The example global files to generate baseflow, runoff and heat budgets are also directly included in the folder **run_VIC**, which you can use directly. The two were implemented as follows:
 <center>**./vicNl –g global_param_Salmon_0.5_flux**</center>
 which generates base flow and runoff
 
 and
 <center>**./vicNl –g global_param_Salmon_0.5_full_data** </center>
 which generates heats budget
 
-PS: The output directories need to be specified in each global parameter files, **global_param_Salmon_0.5_flux** and **global_param_Salmon_0.5_full_data** in this case.
+Note: The output directories need to be specified in each global parameter files, **global_param_Salmon_0.5_flux** and **global_param_Salmon_0.5_full_data** in this case.
```

Comment:
Thanks for catching that! I change the previous line as:
`The example global files to generate baseflow, surface runoff and heat fluxes are also included in the folder **run_VIC**, of which you need to specify the output directory before you use them.`